python/lsst/fgcmcal/fgcmBuildFromIsolatedStars.py

```diff
@@ -388,8 +388,11 @@ def _make_all_star_obs_from_isolated_stars(
             self.config.apertureOuterInstFluxField + "Err",
         ]
 
+        local_background_flag_name = None
```

Comment:
I'm not seeing why this initialization is necessary (as the other use below is also behind a `if self.config.doSubtractLocalBackground:` clause)...